docs/reference/slm/getting-started-slm.asciidoc

```diff
@@ -1,23 +1,31 @@
 [role="xpack"]
 [testenv="basic"]
 [[getting-started-snapshot-lifecycle-management]]
-=== Configure snapshot lifecycle policies
+=== Tutorial: Automate back ups with {slm-init}
 
-Let's get started with {slm} ({slm-init}) by working through a
-hands-on scenario. The goal of this example is to automatically back up {es}
-indices using the <<snapshot-restore,snapshots>> every day at a particular
-time. Once these snapshots have been created, they are kept for a configured
-amount of time and then deleted per a configured retention policy.
+This tutorial demonstrates how to automate daily back ups of {es} indices using an {slm-init} policy.
```

Comment:
> This tutorial demonstrates how to

I thought we typically strayed away form this type of meta language. We could probably rephrase to something like `You can use an {slim-init} policy to` ...

Reply:
Tutorials are the one place the meta-info doesn't really bother me. I'm going to leave it for now & look at the language we use in other places. I'll open another PR to be consistent, one way or another. (With tutorial in the titles, it's not really necessary to repeat "tutorial" in the text for SEO purposes.)